content/en/docs/pipelines/building-pipelines/overview.md

```diff
@@ -0,0 +1,83 @@
++++
+title = "Overview of Pipeline Development"
+description = "Getting started with pipeline development"
+weight = 10          
++++
+
+Kubeflow Pipelines lets you deploy and run portable, scalable machine learning
+(ML) workflows based on Docker containers. You can orchestrate your ML workflow
+as a pipeline using the Kubeflow Pipelines SDK or TensorFlow Extended (TFX).
+
+This document describes how to choose the best option for building your
+pipeline, and provides resources for getting started.
+
+## Selecting a pipelines SDK
+
+At a high level, the Kubeflow Pipelines SDK and TFX take a similar approach
+to defining pipelines. In both cases:
+
+*  A pipeline is a set of components and input parameters. 
+*  Component outputs are called artifacts. Components can accept either
+   pipeline parameters or the artifacts of other components as inputs.
+*  The execution order of pipeline components is a directed acyclic graph of
```

Comment:
```suggestion
*  The execution order of pipeline components is a directed acyclic graph (DAG) of
```

Just for the sake of simplicity if we need to frequently use this term.

Reply:
Let's add this when there is a second instance of DAG on the page.